craft_application/services/lifecycle.py

```diff
@@ -156,12 +156,8 @@ def run(self, step_name: str, part_names: list[str] | None = None) -> None:
             with self._lcm.action_executor() as aex:
                 for action in actions:
                     message = _get_parts_action_message(action)
-                    emit.progress(f"Executing parts lifecycle: {message}")
```

Comment:
Without the `progress()` message, this will look weird once `streaming_brief` mode is enabled because the `progress()` gives the prefix

Reply:
no, that message is shown briefly before things start getting written to the `open_stream()` pipe.
I guess that message should be optional

Reply:
added back! tried and added some more tests about explicitly showing these messages

https://github.com/canonical/craft-application/compare/63183682eab112d64df464cb25cd5bfdb740d8eb..faf6c89772f6db8138cee0c2a23efb17f76768de